pkg/daemon/rpm-ostree.go

```diff
@@ -20,8 +22,21 @@ const (
 	numRetriesNetCommands = 5
 	// Pull secret.  Written by the machine-config-operator
 	kubeletAuthFile = "/var/lib/kubelet/config.json"
+
+	// rpmOstreeVersionMinimum is the minimum required version
+	rpmOstreeVersionMinimum = "2021.14"
```

Comment:
nice idea for sanity check (aside from immediate mcbs use) 1 q from that pov: would hardcoding directly in file might make it go stale over time? how often would we expect this to change?

Reply:
I think we wouldn't change this too often.  In practice, we'll just know out of band when we're using a new feature - it'll have landed in rhel 8.z or fast tracked into OCP.

I honestly don't have a really strong opinion on actually merging this patch.  We can drop it too when we look at merging the mcbs branch.

But for now it'll be a sanity check.

Reply:
makes sense to me!